.gitmodules

```diff
@@ -10,3 +10,6 @@
 [submodule "deps/solana"]
 	path = deps/solana
 	url = https://github.com/solana-labs/solana.git
+[submodule "deps/anchor"]
+	path = deps/anchor
+	url = https://github.com/ngundotra/anchor
```

Comment:
🤔

Reply:
<img width="630" alt="Screen Shot 2022-05-02 at 12 05 30 PM" src="https://user-images.githubusercontent.com/7481857/166267307-88e06c97-e1f5-453b-b1f5-2a6350e36f48.png">

Reply:
I think this is only viable solution